lib/log.h

```diff
@@ -85,6 +85,11 @@ extern void zlog_info(const char *format, ...) PRINTF_ATTRIBUTE(1, 2);
 extern void zlog_notice(const char *format, ...) PRINTF_ATTRIBUTE(1, 2);
 extern void zlog_debug(const char *format, ...) PRINTF_ATTRIBUTE(1, 2);
 
+/* For logs which have error codes associated with them */
+#define zlog_ferr(ferr_id, format, ...)                                        \
+	zlog_err("[EC %d] " format, ferr_id, ##__VA_ARGS__)
```

Comment:
It would be disruptive to to have redine zlog_err to map to automatically extract/add <DIR>_ERR_PRIVILEGES from __FILE__ and obviate the need for touching all instances if zlog_err